docs/modules/getting-started/pages/metrics.adoc

```diff
@@ -0,0 +1,69 @@
+= AeroGear Services Metrics SDK
```

Comment:
Can we drop metrics in name and title? This should be generic getting started that could also have notes about metrics.